### ISSUE TITLE
Unable for OneNote for windows 10

### ISSUE THREAD

**Issue by Margaret-Z** [elvirbrk/NoteHighlight2016#87]
First of all, thanks for your powerful plugin. However,this plugin doesn't seem to work for OneNote for windows10. I can't find this plugin in the trust center in my win10 OneNote after  proper installation. And as is illlustrated on the Microsoft's official website, there are exact differences between OneNote 201x and win10 OneNote. Is there any new version for win10 OneNote?

**Comment by Octav1an** [comment]
Unfortunately, UWP OneNote does not support addins, and as far as I know they will not add it soon. I also prefer the UWP OneNote interface, and I mostly use it, but when I have to paste a code snippet, or to use some other plugin I do it in OneNote 2016. 
One tip for this workflow, is to close OneNote win10 first, then open OneNote 2016, do all of your edits and manually sync before opening OneNote win10.

**Comment by Itsuka-Z** [comment]
> 
> 
> Unfortunately, UWP OneNote does not support addins, and as far as I know they will not add it soon. I also prefer the UWP OneNote interface, and I mostly use it, but when I have to paste a code snippet, or to use some other plugin I do it in OneNote 2016.
> One tip for this workflow, is to close OneNote win10 first, then open OneNote 2016, do all of your edits and manually sync before opening OneNote win10.

Thank you so much for your useful advice. Apart from your method,I have found another way to highlight the code in win10 OneNote. Just by using the NotePad++ and its convenient plugin(can be downloaded easily on the Internet) ,the code can be copied with highlight  in your Win 10 OneNote.

**Comment by Downright-Development** [comment]
@Itsuka-Z What plugin do you use for Notepad++ for this? NppExport? I too use the UWP app.

**Comment by Jediguy** [comment]
@amp4045 Not sure if you still need it, but I figured I'd leave a workaround I just found after reading this. I too use UWP OneNote and came across this plugin only to find that it sadly didn't work. I didn't have Notepad++ installed but I already had a vs code window open. Just copy and pasted it there then was able to copy paste that to OneNote and the formatting stayed the same and is super clean. So if anyone else stumbles upon this I hope this helps.
